File: python/copass-management/src/copass_management/tools/revoke_sandbox_connection.py
from __future__ import annotations

from typing import Any, Dict, TYPE_CHECKING

if TYPE_CHECKING:
    from copass_management.registrar import ToolContext


async def revoke_sandbox_connection(
    ctx: "ToolContext", input: Dict[str, Any],
) -> Any:
    connection_id = str(input["connection_id"])
    return await ctx.client.sandbox_connections.revoke(
        ctx.sandbox_id, connection_id,
    )

File: typescript/packages/management/src/tools/revoke_sandbox_connection.ts
import type { ToolContext, ToolHandler } from '../registrar.js';

export const revokeSandboxConnection: ToolHandler = async (
  ctx: ToolContext,
  input: Record<string, unknown>,
) => {
  const connectionId = String(input.connection_id);
  return ctx.client.sandboxConnections.revoke(ctx.sandboxId, connectionId);
};
